AUTHORS

```diff
@@ -70,6 +84,7 @@ Andrew Williams <williams.andrew@gmail.com>
 Andrews Medina <andrewsmedina@gmail.com>
 Andrey Petrov <andrey.petrov@shazow.net>
 Andrey Stolbovsky <andrey.stolbovsky@gmail.com>
+André Martins <aanm90@gmail.com>
 André Martins <martins@noironetworks.com>
```

Comment:
This is an example of what `.mailmap` is for -- these are probably the same contributor, so the `.mailmap` can be used to squash them into a single entry (after doing a little basic digging to verify that they really are the same contributor).